dmscripts/export_framework_results_reasons.py

```diff
@@ -173,14 +173,14 @@ def get_questions_numbers_from_framework(framework_slug, content_loader):
 
 
 def export_suppliers(
-        client,
-        framework_slug,
-        content_loader,
-        output_dir,
-        declaration_definite_pass_schema,
-        declaration_baseline_schema=None,
-        supplier_ids=None,
-        ):
+    client,
+    framework_slug,
+    content_loader,
+    output_dir,
+    declaration_definite_pass_schema,
+    declaration_baseline_schema=None,
+    supplier_ids=None,
+):
```

Comment:
So many lines...

Reply:
Yeah this function probably does a bit too much...